src/components/buttons/LaunchModalBtn.tsx

```diff
@@ -1,16 +1,18 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
 import { Button } from '@/components/ui/button';
 import { TbCirclePlus } from 'react-icons/tb';
 
 interface LaunchModalBtnProps {
-  title: string,
+  onClick: () => void;
+  title: string;
 }
-export const LaunchModalBtn = ({title}: LaunchModalBtnProps) => {
+export const LaunchModalBtn = forwardRef<
+  HTMLButtonElement,
+  LaunchModalBtnProps
+>(({ title, onClick }, ref) => {
   return (
-    <Button className="gap-2">
-      <TbCirclePlus /> {title}
+    <Button ref={ref} onClick={onClick} className="gap-2">
+      <TbCirclePlus size={20} aria-label="Add Icon" /> {title}
```

Comment:
is this aria-label helpful or more confusing when used with a text reader, have you checked it?
If I correctly understand how aria works it now says "Add icon add a new pet" for this button... but I'm not sure.